client/src/components/imagine25/pages/Game.js

```diff
@@ -1,18 +1,123 @@
+  //Random score that will be generated for both teams
+  const totalUserScore = Math.floor(Math.random() * 1000 + 500);
+
+  const userScore = Math.floor(
+    Math.random() * (totalUserScore * 0.6) + totalUserScore * 0.2,
+  );
+  const teammateScore = totalUserScore - userScore;
+
+  /*opponent score will always be less than user score, but never less than 475
+      This is done so that the game seems realistically close*/
+  const totalOpponentScore = Math.floor(
+    Math.random() * (totalUserScore * 0.8) + totalUserScore * 0.2,
+  );
+
+  // Ensure a fair distribution between opponents
+  const opponentScore1 = Math.floor(
+    Math.random() * (totalOpponentScore * 0.6) + totalOpponentScore * 0.2,
+  );
+  const opponentScore2 = totalOpponentScore - opponentScore1;
+
+  return (
+    <>
+      <h3 className="tw-title text-center">Game Outcome</h3>
+
+      <div className="tw-grid tw-grid-cols-2 tw-pt-8 tw-justify-center">
+        <div className="tw-my-20 tw-body-text tw-mx-auto">
+          <div className="tw-font-bold">
+            Overall Team Score: {userScore + teammateScore}
+          </div>
+          <div>Your Score: {userScore}</div>
+          <div>Your Teammate Score: {teammateScore}</div>
+        </div>
+
+        <div className="tw-my-20 tw-body-text tw-mx-auto">
+          <div className="tw-font-bold ">
+            Overall Opponent Score: {totalOpponentScore}
+          </div>
+          <div>Opponent 1 Score: {opponentScore1}</div>
+          <div>Opponent 2 Score: {opponentScore2}</div>
+        </div>
+      </div>
+
+      <Button
+        className="btn btn-primary text-black btn-xl text-uppercase tw-m-3"
+        onClick={nextPage}
+      >
+        Analyze Game
+      </Button>
+    </>
+  );
+};
+//I hate that this validation is required
+ScorePage.propTypes = {
+  nextPage: PropTypes.func,
+};
+
+const Analysis = () => {
+  const [content, setContent] = useState(null);
+
+  //until the userID is grabbed, the page will techincally be blank until the useeffect activates
+  useEffect(() => {
+    const fetchContent = async () => {
+      //yoink that user data
+      const user = await ImagineService.getUserByID(
+        sessionStorage.getItem("userID"),
+        25,
+      );
+
+      //using map instead of "code smell" switch statment ft - Professor Bobby (st.Jaques or something like that)
+      const text = {
+        experiential:
+          "Sorry, your shirt is " +
+          user.avatar.clotheColor +
+          ", we simply cannot tolerate such awful choices and you and your teammte are both are disqualified.",
+        expression:
+          "Sorry, your teammate's shirt is " +
+          user.teammateAvatar.clotheColor +
+          ", we simply cannot tolerate such awful choices. You and your teammte are both are disqualified.",
+        control:
+          "Congrats on winning! You may collect a prize after completeting the post survery for being so awesome sauce.",
```

Comment:
I assume this is just placeholder text until we get the final versions from krutz right?😆

Reply:
Another thought: I wonder if we want to store the disqualification reason in the database with each user instance? Or can we just assume that since we record the user's section (experiential, expression, or control), we always know what the disqualification reason was